aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,196 @@
+        origin.get("DomainName", "")
+        for origin in dist.get("Origins", {}).get("Items", [])
+    ]
+
+    if any("app." in domain for domain in origin_domains):
+        print(f"Skipping app distribution: {dist['Id']} (origins: {origin_domains})")
+        return None
+
+    if dist.get("IsStagingDistribution", False):
+        print(
+            f"Found staging distribution: {dist['Id']} "
+            f"(IsStagingDistribution: true, origins: {origin_domains})"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)